examples/todomvc/src/router.rs

```diff
@@ -0,0 +1,62 @@
+//! A simple `#`-fragment router.
+
+use crate::todos::Todos;
+use crate::utils;
+use crate::visibility::Visibility;
+use dodrio::VdomWeak;
+use futures::prelude::*;
+use std::str::FromStr;
+use wasm_bindgen::prelude::*;
+use wasm_bindgen::JsCast;
+
+/// Start the router.
+pub fn start(vdom: VdomWeak) {
+    // Callback fired whenever the URL's hash fragment changes. Keeps the root
+    // TODOs collection's visibility in sync with the `#` fragment.
```

Comment:
now this is tricky

It's a comment that starts with "TODO"

not a TODO comment